package.json

```diff
@@ -60,12 +60,14 @@
     "electron-squirrel-startup": "^1.0.0",
     "electron-winstaller": "^2.3.1",
     "end-of-stream": "^1.1.0",
+    "fs-blob-store": "^5.2.1",
```

Comment:
Unused dep, it looks like.

Reply:
ah yeah it should use safe-fs-blob-store